src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+body {
+  color: $main-color;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding:  32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__image {
+    margin-inline: 3px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+   display: flex;
+   flex-direction: column;
+   justify-content: space-between;
+  }
+
+  &__title {
+    margin: 0;
+    margin-top: 38px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 4px 0 16px;
+
+    font-size: 10px;
+
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    text-align: right;
```

Comment:
it's extra, no? You do it with `justify-content: space-between;`
So, remove it & check other cases